olive/workflows/run/config.py

```diff
@@ -24,6 +24,23 @@
 logger = logging.getLogger(__name__)
 
 
+class PassConfig(ConfigBase):
```

Comment:
Can you use a different name like `PassModuleConfig` or something since we already have similar named configs like `RunPassConfig`, `FullPassConfig`, `PassConfigParam`?

Reply:
In my opinion, this is the most appropriate name, especially since the other is named with prefix 'Run'. I would argue that the other classes should be named with the same prefix because they relate to the 'run'.

Regardless, how about `PassPackageConfig` since files are considered package in Python (borrowing the convention from `importlib.import_module` API where the second parameter is named `package`.)